src/main/java/ExceptionHandling/InvalidTaskIndexException.java

```diff
@@ -0,0 +1,4 @@
+package ExceptionHandling;
+
+public class InvalidTaskIndexException extends Exception{
+}
```

Comment:
Advisable to have a uniform spacing strategy, to enhance code readability.

```suggestion
public class InvalidTaskIndexException extends Exception {
}
```